airflow-core/docs/core-concepts/dags.rst

```diff
@@ -829,3 +829,40 @@ if it fails for ``N`` number of times consecutively.
 we can also provide and override these configuration from DAG argument:
 
 - ``max_consecutive_failed_dag_runs``: Overrides :ref:`config:core__max_consecutive_failed_dag_runs_per_dag`.
+
+Deadline Alerts
+---------------
+
+.. versionadded:: 3.1
+
+Deadline Alerts allow you to set time thresholds for your DAG runs and automatically respond when those
+thresholds are exceeded. You can set deadlines relative to a fixed datetime, use one of the available
+calculated references (like DAG queue time or start time), or implement your own custom reference.
+When a deadline is exceeded, it triggers a callback which can notify you or take other actions.
+
+Here's a simple example using the existing email Notifier:
```

Comment:
Unless you have good reason to keep it capitalized?

```suggestion
Here's a simple example using the existing email notifier:
```

Reply:
Do we not use the proper noun there?   I'll see if I can find precedent in other doc pages, but I think we usually capitalize things lie Operator, Sensor, Notifier, etc.  I'll double check.

Reply:
I like capitalizing the proper noun here, FWIW